app/src/main/res/layout/activity_session_feedback.xml

```diff
@@ -0,0 +1,315 @@
+                android:paddingEnd="@dimen/spacing"
+                android:paddingLeft="@dimen/spacing"
+                android:paddingRight="@dimen/spacing"
+                android:paddingStart="@dimen/spacing"
+                android:paddingTop="@dimen/spacing_small"
+                android:text="@string/session_feedback_knowledgeable"
+                android:textAlignment="viewStart" />
+
+            <io.github.droidkaigi.confsched.widget.NumberRatingBar
+                android:id="@+id/speaker_feedback_bar"
+                style="@style/FeedbackNumberRatingBar"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginEnd="@dimen/spacing_small"
+                android:layout_marginLeft="@dimen/spacing_small"
+                android:layout_marginRight="@dimen/spacing_small"
+                android:layout_marginStart="@dimen/spacing_small"
+                android:maxHeight="?android:listPreferredItemHeightSmall"
+                android:minHeight="?android:listPreferredItemHeightSmall"
+                android:paddingTop="@dimen/spacing_small" />
+
+            <FrameLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:paddingBottom="@dimen/spacing"
+                android:paddingEnd="@dimen/spacing"
+                android:paddingLeft="@dimen/spacing"
+                android:paddingRight="@dimen/spacing"
+                android:paddingStart="@dimen/spacing"
+                android:paddingTop="@dimen/spacing_xsmall">
+
+                <TextView
+                    style="@style/FeedbackQuestionGuideText"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_gravity="start|left"
+                    android:text="@string/session_feedback_poor_knowledge" />
+
+                <TextView
+                    style="@style/FeedbackQuestionGuideText"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_gravity="end|right"
+                    android:text="@string/session_feedback_outstanding_knowledge" />
+
+            </FrameLayout>
+
+            <!-- Additional Comment -->
+            <TextView
+                style="@style/FeedbackTitle"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:gravity="center_vertical"
+                android:minHeight="?android:listPreferredItemHeightSmall"
+                android:paddingEnd="@dimen/spacing"
+                android:paddingLeft="@dimen/spacing"
+                android:paddingRight="@dimen/spacing"
+                android:paddingStart="@dimen/spacing"
+                android:paddingTop="@dimen/spacing_small"
```

Comment:
I guess it is better to put these attributes to `@style/FeedbackTitle`. How do you think?